src/Components/Components/test/BindConverterTest.cs

```diff
@@ -298,6 +298,77 @@ public void FormatValue_TypeConverter()
         Assert.Equal(expected, actual);
     }
 
+    [Fact]
+    public void TryConvertTo_Guid_Valid()
+    {
+        // Arrange
+        var expected = Guid.NewGuid();
```

Comment:
Super nit, if this fails for some arbitrary input, we would never find out what the input value was. Maybe we should invert  lines 312 and 313 or just use a fixed Guid.

Reply:
Good point, though I'll be pretty astonished if some GUIDs work and others don't. Swapped the two lines around as suggested anyway :)